src/app/index.module.js

```diff
@@ -25,9 +26,11 @@ const App = angular.module(
   "angularApp", [
     // plugins
     uiRouter,
+    toastr,
```

Comment:
Useless because You include it on line 33

Reply:
Visa versa, 29 string is right 33 wrong. Fixed it.